app/code/Magento/LoginAsCustomerGraphQl/Model/LoginAsCustomer/CreateCustomerToken.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\LoginAsCustomerGraphQl\Model\LoginAsCustomer;
+
+use Magento\Customer\Model\CustomerFactory;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Integration\Model\Oauth\TokenFactory;
+use Magento\Store\Api\Data\StoreInterface;
+use Exception;
+
+/**
+ * Create customer token from customer email
+ *
+ * Class CreateCustomerToken
```

Comment:
Please, remove the unnecessary comments. These comments are auto-generated and the only purpose is to add extra lines to your code. 

```suggestion

```

Reply:
Removed